pages/features/auto-responder.vue

```diff
@@ -174,6 +177,12 @@ export default {
       ],
     }
   },
+  async asyncData() {
+    let randomTestimonials = await allTestimonials
+    const randIndex = Math.floor(Math.random() * (randomTestimonials.length - 2))
+    randomTestimonials = randomTestimonials.slice(randIndex,  randIndex + 2);
+    return {randomTestimonials}
+  },
```

Comment:
Can this be a part of mixin?

Reply:
tried using mixins, but things will get more complicated as I'm getting data as promise so first I will need to resolve it in all pages and than pass the data to testimonial component, and that will be basically the same thing @ankitsinghaniyaz